app/payload/control_freak/flight-software/src/flight.h

```diff
@@ -0,0 +1,85 @@
+#ifndef FREAK_FLIGHT_H
+#define FREAK_FLIGHT_H
+#include "f_core/flight/c_phase_controller.h"
+
+#include <array>
+
+// Boost
+// IMU
+static constexpr float imuBoostThresholdMPerS2 = 5 * 9.8;  // m/s^2
+static constexpr uint32_t imuBoostTimeThreshold = 250;     // ms
```

Comment:
nit. Can we just write millis here just to be explict. The m/s and ms right below it may look unclear to other readers